src/prefect/client/utilities.py

```diff
@@ -0,0 +1,52 @@
+import time
+
+from prefect.client.orion import get_client
+from prefect.utilities.asyncutils import sync_compatible
+
+
+@sync_compatible
+async def run_deployment(
```

Comment:
Not necessary for the issue itself but FYI you could expose a `scheduled_start_time: datetime.datetime` field here and then pass `state = schemas.states.Scheduled(scheduled_time=scheduled_start_time)` to the `create_flow_run_from_deployment` client method in order to expose the ability to optionally schedule the run for the future.

Reply:
ah good catch, this is a good idea